mne/chpi.py

```diff
@@ -149,6 +150,115 @@ def head_pos_to_trans_rot_t(quats):
     return translation, rotation, t
 
 
+def _apply_quat(quat, pts, move=True):
+    """Apply MaxFilter-formatted head position parameters to points."""
+    trans = np.concatenate(
+        (quat_to_rot(quat[:3]),
+         quat[3:][:, np.newaxis]), axis=1)
+
+    return(apply_trans(trans, pts, move=move))
+
+
+def extract_head_pos_ctf(raw, gof_limit=0.98):
+    """Extract MaxFilter-formatted head position parameters from ctf dataset.
+
+    Parameters
+    ----------
+    raw : instance of raw
+        Raw data with cHPI information. HLC00 channels
+    gof_limit : float
+        Minimum goodness of fit to accept.
+
+    Returns
+    -------
+    pos : array, shape (N, 10)
+        The position and quaternion parameters from cHPI fitting.
+
+    See Also
+    --------
+
+    Notes
+    -----
+    CTF continuous head monitoring stores the x,y,z location (m) of each chpi
+    coil as separate channels in the dataset.
+    HLC001[123]-* - nasion
```

Comment:
These `*` will probably cause Sphinx to complain. You can either escape them with `\` (since it's in a string it should actually be `\\` or use an r'' string) or change the nomenclature.